addons/pos_self_order/models/res_config_settings.py

```diff
@@ -93,6 +94,9 @@ def _generate_single_qr_code(self, url):
         )
         qr.add_data(url)
         qr.make(fit=True)
+        if format == "SVG":
+            image_factory = qrcode.image.svg.SvgImage
+            return qr.make_image(fill_color="black", back_color="transparent", image_factory=image_factory)
         return qr.make_image(fill_color="black", back_color="transparent")
```

Comment:
Since this method is only used here, you can maybe return both SVG and image here, to avoid duplicate bellow:
```suggestion
image_factory = qrcode.image.svg.SvgImage
return {
    'svg': qr.make_image(fill_color="black", back_color="transparent", image_factory=image_factory),
    'png': qr.make_image(fill_color="black", back_color="transparent"),
}
```

Reply:
:+1 done !